### ISSUE TITLE
fig not working in vs code from today

### ISSUE THREAD

**Issue by rama-kairi** [withfig/fig#1978]
### Checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and replicated the issue again

### Operating system

macOS 13.0.0 (22A380)

### Expected behaviour

![Uploading image.png…]()

### Actual behaviour

![Uploading image.png…]()

### Steps to reproduce

![image](https://user-images.githubusercontent.com/64131361/199752509-89376246-9fce-46f6-95f2-99485d7bc8ab.png)

### Environment

```yaml
fig-details:
  - 2.7.2
hardware-info:
  - model: 
  - model-id: 
  - chip-id: Apple M1 Pro
  - cores: 8
  - mem: 16.00 GB
os-info:
  - macOS 13.0.0 (22A380)
environment:
  - shell: /opt/homebrew/Cellar/fish/3.5.1/bin/fish
  - terminal: vscode
  - cwd: /Users/ramanandakairi/work/simplyjet-v2-backend
  - exe-path: /opt/homebrew/bin/fig
  - install-method: brew
  - env-vars:
    - FIG_SET_PARENT: 
    - TERM: xterm-256color
    - SHELL: /opt/homebrew/bin/fish
    - PATH: /Users/ramanandakairi/Library/pnpm:/Users/ramanandakairi/.pyenv/shims:/Users/ramanandakairi/.cargo/bin:/Users/ramanandakairi/.pyenv/bin:/opt/homebrew/bin:/Users/ramanandakairi/.bun/bin:/opt/homebrew/sbin:.:/opt/homebrew/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Users/ramanandakairi/Library/pnpm:/Users/ramanandakairi/.cargo/bin:/opt/homebrew/sbin:.:/Users/ramanandakairi/.fig/bin:/Users/ramanandakairi/.local/bin:/Users/ramanandakairi/go/bin:/Users/ramanandakairi/go/bin
    - FIG_PARENT: 
    - FIG_HOSTNAME: Rama-Mac
    - FIG_PID: 12255
    - FIGTERM_SESSION_ID: af1bb8e5-19e2-4f7c-aab3-192cba8b6ac7
    - FIG_TERM: 2.7.2
```

**Comment by mschrage** [comment]
Hey @rama-kairi, it looks like your images didn't upload properly. Can you try to upload them again?

**Comment by vladpalkanov** [comment]
The same for me, it works in Iterm terminal, for instance, but it doesn't work in vscode terminal.

Fig vscode extension version: 0.0.7 / Last updated
04/11/2022, 11:45:02

```yaml
fig-details:
  - 2.7.4
hardware-info:
  - model: 
  - model-id: 
  - chip-id: Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz
  - cores: 8
  - mem: 32.00 GB
os-info:
  - macOS 13.0.0 (22A380)
environment:
  - shell: /usr/local/Cellar/zsh/5.8_1/bin/zsh
  - terminal: vscode
  - cwd: /Users/uladzislau_palkanau/study/p2p/chat
  - exe-path: /usr/local/bin/fig
  - install-method: unknown
  - env-vars:
    - FIGTERM_SESSION_ID: 775084a9-8259-416e-a348-85edba38b53b
    - FIG_HOSTNAME: EPPLWROW027A
    - FIG_PARENT: 045b28eb-ee38-42e8-96e4-d405e611b81b
    - FIG_PID: 31303
    - FIG_SET_PARENT: 775084a9-8259-416e-a348-85edba38b53b
    - FIG_TERM: 2.7.4
    - PATH: /usr/local/sbin:/Users/uladzislau_palkanau/.nvm/versions/node/v18.10.0/bin:/Users/uladzislau_palkanau/.nvm/versions/node/v16.17.1/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/sbin:/Users/uladzislau_palkanau/.nvm/versions/node/v16.17.1/bin:/Users/uladzislau_palkanau/.nvm/versions/node/v16.15.0/bin:/Users/uladzislau_palkanau/.cargo/bin:/Users/uladzislau_palkanau/.fig/bin:/Users/uladzislau_palkanau/.local/bin:/usr/local/bin
    - SHELL: /usr/local/bin/zsh
    - TERM: xterm-256color
```

**Comment by vladpalkanov** [comment]
<img width="533" alt="Screenshot 2022-11-04 at 12 07 50" src="https://user-images.githubusercontent.com/8327420/199958550-8dbf9fb2-d29d-4d3d-8ecf-f72d70b296cb.png">
<img width="570" alt="Screenshot 2022-11-04 at 12 08 29" src="https://user-images.githubusercontent.com/8327420/199958674-df7e4da7-11df-428f-8524-b42045ee130a.png">

**Comment by andy-blum** [comment]
I noticed similar behavior to this issue, but after seeing #1975, the issue for me was the popover list wasn't appearing _only while VS code was in fullscreen mode_. Taking it out of fullscreen made the popover appear back in the right spot for me. While not a "fix" per se, it may re-enable functionality for you.

**Comment by mschrage** [comment]
@andy-blum thanks for clarify this. I think the issue is ultimately with full screen apps rather than VSCode specifically

**Comment by vladpalkanov** [comment]
@andy-blum  @mschrage yes, the popover does not appear in the  fullscreen  Iterm2 too

**Comment by mschrage** [comment]
We have a fix for this that should be released in the next day or so.

**Comment by mschrage** [comment]
Fix is live on beta: `2.7.6-beta.2`. Run `fig settings app.beta true` to try it out now.

**Comment by vladpalkanov** [comment]
> Fix is live on beta: `2.7.6-beta.2`. Run `fig settings app.beta true` to try it out now.

now it works for me

**Comment by mschrage** [comment]
2.7.6 is live! To upgrade to the latest version, run `fig update`.
